src/main/java/com/contrastsecurity/maven/plugin/InstallAgentContrastMavenMojo.java

```diff
@@ -27,6 +27,8 @@ public void execute() throws MojoExecutionException {
 
         getLog().info("Agent downloaded.");
 
+        String serverName2;
```

Comment:
What is this variable representing?

Reply:
This variable is for the server name that is retrieved from TS. I renamed it to `retrievedServerName`